include/nuttx/signal.h

```diff
@@ -696,6 +698,28 @@ void nxsig_cancel_notification(FAR struct sigwork_s *work);
 #  define nxsig_cancel_notification(work)
 #endif
 
+#else
+
+#  define nxsig_ismember(set,sig)          -EINVAL
+#  define nxsig_addset(set,sig)            -EINVAL
```

Comment:
Please remove all dummy function which always report error, since  we need change the runtime error to compile error and then skip compiling the broken function directly.